### ISSUE TITLE
Builds with this target are not viable

### ISSUE THREAD

**Issue by dwimberger** [ARMmbed/target-efm32pg-stk-gcc#1]
I have tried to use this target to build a firmware for the development kit, but the firmware does not work on the development kit.

The example for getting started:
https://docs.mbed.com/docs/getting-started-mbed-os/en/latest/FirstProjectmbedOS/

```
yotta init
yotta target efm32pg-stk-gcc
yotta install mbed-drivers
yotta build
```

The binary is built, it can be loaded onto the device only via Flash Tool in the Simplicity Studio (Mbed ignores it), but there is absolutely no reaction from the board and I have no clue how to figure out what is going wrong.

Any idea?

**Comment by stevew817** [comment]
Duplicate of https://developer.mbed.org/questions/69306/Is-there-any-actually-working-sample-for/

**Comment by dwimberger** [comment]
Steven's fix on mbed-hal-efm32pg@1.2.0 fixes the problem. 

The resulting binary is now loaded and executed. Got blinky sample up and running 👍
